tests/src/Kernel/Entity/SelectionHandlerTest.php

```diff
@@ -117,8 +117,6 @@ protected function setUp() {
    */
   public function testSelectionHandler() {
     $this->assertEquals(get_class($this->selectionHandler->getSelectionHandler()), 'Drupal\node\Plugin\EntityReferenceSelection\NodeSelection');
-    $this->assertEquals($this->selectionHandler->getConfiguration('handler'), 'default:node');
-    $this->assertEquals($this->selectionHandler->getConfiguration('target_type'), 'node');
```

Comment:
We get that from the class? It just means we have to keep the `getConfiguration` method purely for the tests, which is not cool. This is kind of re-testing the core entity selection handler, which I don't think we need to do.